python/mlc_llm/quantization/group_quantization.py

```diff
@@ -602,6 +597,7 @@ def from_linear(cls, src: nn.Linear, config: GroupQuantize) -> "GroupQuantizeLin
         if "shard_strategy" in src.weight.attrs:
             shard = src.weight.attrs["shard_strategy"]
             apply_sharding(shard, f"{shard.name}_q_weight", quantized_linear.q_weight)
+            # TODO(@team): upstream does not require this condition.
```

Comment:
cc. @csullivan @JosephTheOctonaut @vinx13

Reply:
we can remove it and all fp8 contents in this file